readme.md

```diff
@@ -91,14 +105,13 @@ Here are some alternate ways of looping
 * `.times` (called on a number)
   * runs the loop a specific of times
```

Comment:
I like the idea, just not sure if timing would allow.  Any specific examples come to mind?